.storybook/main.ts

```diff
@@ -0,0 +1,16 @@
+import type { StorybookConfig } from '@storybook/experimental-nextjs-vite'
+
+const config: StorybookConfig = {
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
```

Comment:
```suggestion
  stories: ['../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
```

re: my other comment